src/main/java/nl/tudelft/ti2206/group9/audio/SoundEffectPlayer.java

```diff
@@ -37,6 +42,7 @@ public SoundEffectPlayer(final String soundPath) {
         if (State.isSoundEffectsEnabled()) {
             initializeAudio(path);
         }
+        this.setVolume(DEFAULT_VOLUMELEVEL);
```

Comment:
Shouldn't the volume be set to `State.getVolume()`? (Same goes for SoundtrackPlayer)

<review, logic